src/VirtualClient/VirtualClient.Dependencies/Packaging/WgetPackageInstallation.cs

```diff
@@ -149,6 +149,15 @@ protected override async Task ExecuteAsync(EventContext telemetryContext, Cancel
 
                                     await this.packageManager.ExtractPackageAsync(downloadedPackagePath, installationPath, cancellationToken, archiveType);
                                 }
+                                else
+                                {
+                                    // This section is for the standalone files which are not zipped. The file will be copied to a location in installationPath and then
+                                    // file at original download path is deleted as in above case for zip file.
+                                    string installationFilePath = this.GetPackagePath(this.PackageName, Path.GetFileName(this.PackageUri.ToString()));
```

Comment:
Could we add a unit test for the path and copy for this scenario? Also let's not imply a file downloaded is an installFilePath. In addition, in comment at line 155 you mentioned the file at original path would be deleted? In this case you are not deleting it?

Reply:
Added UnitTests for this part as suggested.
Modified the variable name for better understanding.
The original downloaded file is being deleted at the end of function at line # 177